### PR TITLE
Fix wrong EIP and add links

### PR DESCRIPTION
**Pull request by anxolin**
# Summary

Fixes the issue raised here https://gnosisinc.slack.com/archives/CPZA1AGKY/p1636037095201900?thread_ts=1636036992.200200&cid=CPZA1AGKY

Basically, we wrote EIP712 when we meant EIP1271 for SC signing

image

# To Test
Verify the changes

### CODE REVIEW

File: src/custom/pages/Faq/index.tsx [hunk 1]
```diff
@@ -434,8 +437,12 @@ export default function Faq() {
             solutions.
           </p>
           <p>
-            In the future, the protocol might provide EIP712 support for off-chain signing also for smart contracts,
-            making gas-less trading possible also for smart contracts.
+            In the future, the protocol might provide{' '}
+            <ExternalLink href="https://eips.ethereum.org/EIPS/eip-1271" target="_blank" rel="noopener noreferrer">
+              EIP-1271
+            </ExternalLink>{' '}
```
Comment (nlordell) [inline src/custom/pages/Faq/index.tsx:443]:
```suggestion
            </ExternalLink>{' '}
```

JSX is funny. I remember having an incredibly hard time having an inline `<tag>` followed by a line-break to not add whitespace in plain HTML (had to `<!-- comment -->` out the line-break), and it looks like in JSX you have the opposite issue :stuck_out_tongue:.

File: src/custom/pages/Faq/index.tsx [hunk 2]
```diff
@@ -414,7 +414,10 @@ export default function Faq() {
             <LinkScrollable href={'#what-is-presign'}>pre-sign</LinkScrollable>.
           </p>
           <p>
-            Regular, non Smart Contract wallets (called EOA) require the wallet to support offchain-signing (EIP712
+            Regular, non Smart Contract wallets (called EOA) require the wallet to support offchain-signing (
+            <ExternalLink href="https://eips.ethereum.org/EIPS/eip-712" target="_blank" rel="noopener noreferrer">
```
Comment (nlordell) [inline src/custom/pages/Faq/index.tsx:418]: Question out of curiosity, what is the difference between `ExternalLink` and `a`, and why doesn't `ExternalLink` already add things like `target="_blank" rel="noopener noreferrer"`?

**Review by nlordell — APPROVED** [review]
LGTM

File: src/custom/pages/Faq/index.tsx (hunk 2 above)
Comment (anxolin) [reply src/custom/pages/Faq/index.tsx:418]: It has to do with analytics and styles. We track the clicks user do on those links. `ExternalLink` adds already `target="_blank"` by default that's a good point, its redundant.

Comment (anxolin) [reply src/custom/pages/Faq/index.tsx]: I used this PR to simplify the links and remove the redundant tags

**Review by alfetopito — APPROVED** [review]

**Comment by elena-zh** [comment]
Hey @anxolin , 
It would be great to make 'Pre-sign' start with the capital letter
image

In addition, I'm sure that it is exactly not related to this PR, but may be it would be nice to update this info?
image

**Comment by anxolin** [comment]
> It would be great to make 'Pre-sign' start with the capital letter

Done!


> In addition, I'm sure that it is exactly not related to this PR, but may be it would be nice to update this info?

Will do in another PR, cause I see some discrepancy. But thanks for reminding us!!

**Pull request merged**